src/views/_screen-xs-max.scss

```diff
@@ -10,3 +10,58 @@
 		width: 100%;
 	}
 }
+
+/* Responsive tables - for Tablet Cards - optional */
+.provisional {
+	&.gc-table {
+	.tblt-crds {
```

Comment:
Same question here but for the `.tblt-crds` class (and even `.left-xs`). Is there a Javascript plugin that does all the transformations here and that is not included in this PR or does a publisher/web author would need to put all the classes at the right spots?

A working example would definitely help seeing everything working together. Could you please add an **.hbs** file that contains such example in the following folder? https://github.com/wet-boew/GCWeb/tree/master/site/pages/component

Reply:
This is a PR draft - I'm still working on it, there's likely not even going to be a need for the tablet cards because my initial breakpoints in my prototype were different than what GCWeb actually uses.

I am not planning a javascript plugin at this time.

I do have the working example .hbs file - but it's incomplete